golem_messages/message/base.py

```diff
@@ -77,8 +77,8 @@ def verify_version(msg_version):
         )
 
 
-def _verify_slot_type(value, class_):
-    if not isinstance(value, (class_, type(None))):
+def _verify_slot_type(value, class_, allow_none=False):

```

Comment:
How about naming it with the `is_` convention, as it is bool, ie. `is_none_allowed`? Just a proposal.

Reply:
I think `allow_none` is pretty obvious in this context ... and `is_none_allowed` seems like a name of a flag (property) rather than a parameter to a function...